### ISSUE TITLE
Cannot read property 'startDiscovery' of null

### ISSUE THREAD

**Issue by woflmao** [kenjdavidson/react-native-bluetooth-classic#261]
- Device: [Expo dev client]
- React Native version: [0.71.8]
- RN Bluetooth Classic version: [^1.60.0-rc.28]
- with-rn-bluetooth-classic: [^1.0.5]

**Describe the bug**
when I try to use RNBluetoothClassic I get the following:  {"_eventEmitter": {"_nativeModule": {}}, "_nativeModule": null} and when I try to access methods I get this error: TypeError: Cannot read property 'startDiscovery' of null.
This applies for all methods on RNBluetoothClassic

**To Reproduce**
1. Install rnbluetoothclassic and with-rn-bluetooth-classic
2. import RNBluetoothClassic and attempt to use it or access methods on it

**Expected behavior**
I expected to access the methods to start using the package

**Comment by kenjdavidson** [comment]
There are a number of discussions and issues open with regards to expo already.

TLDR;
- you have to export expo
- you have to pay for the professional expo environment to load native modules
